animation_nodes/ui/node_menu.py

```diff
@@ -201,6 +201,7 @@ def draw(self, context):
         layout.separator()
         insertNode(layout, "an_TextBlockReaderNode", "Block Reader")
         insertNode(layout, "an_TextBlockWriterNode", "Block Writer")
+        insertNode(layout, "an_FilterBlendDataListByNameNode", "Filter Text Block List", {"dataType" : repr("Text")})
```

Comment:
Shouldn't this be `Text Block` instead of `Text`?

Reply:
I also thought that but in the filter node, the data type is written as "Text Block".

Reply:
I mean it should be `repr("Text Block")`. `Text` is not supported by the node as far as I know.

Reply:
Sorry, I misunderstood. Yes, it is `Text Block`. However, it is also working with `Text`.

Reply:
It is fixed.

Reply:
Good. But I don't think it works with `Text`, str doesn't have a name attribute.

Reply:
The node is not working properly. But it does not show any error,
![Screenshot from 2019-12-29 17-51-12](https://user-images.githubusercontent.com/30294746/71556884-fd6b0680-2a63-11ea-8520-d64fc6bc3e8e.png)